From 1cbaeef88f7235751a055f0df4947fb38bb7ac13 Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Wed, 23 Mar 2022 09:54:01 +0200
Subject: [PATCH] [SW-77829] rom: reorder status bits

Reorder boot desc status bits as per order of execution.

Change-Id: I8fd0f06cc736e39791c2b14e1896fd68c4c74dfc
---

diff --git a/include/arcboot.h b/include/arcboot.h
index 54bf66d..c218302 100644
--- a/include/arcboot.h
+++ b/include/arcboot.h
@@ -103,8 +103,8 @@
 	PROGRESS_INIT_SPI_BOOT_DESC_UNALIGNED_DEST = 5,
 	PROGRESS_INIT_SPI_BOOT_DESC_UNALIGNED_IMG_SIZE = 6,
 	PROGRESS_INIT_SPI_BOOT_DESC_DMA_ERROR = 7,
-	PROGRESS_INIT_SPI_BOOT_DESC_FAIL = 8,
-	PROGRESS_INIT_SPI_BOOT_DESC_DMA_FINISH = 9,
+	PROGRESS_INIT_SPI_BOOT_DESC_DMA_FINISH = 8,
+	PROGRESS_INIT_SPI_BOOT_DESC_FAIL = 9,
 	PROGRESS_INIT_EFUSE_SPI = 10,
 	PROGRESS_REVOKE_LOC = 11,
 	PROGRESS_REVOKE_LOC_0 = 12,
